lonestar/CMakeLists.txt

```diff
@@ -82,6 +74,7 @@ if(GALOIS_ENABLE_DIST)
 
   set(RESULT_CHECKER ${PROJECT_SOURCE_DIR}/scripts/result_checker.py)
   cmake_host_system_information(RESULT HOSTNAME QUERY HOSTNAME)
+  file(MAKE_DIRECTORY "${CMAKE_BINARY_DIR}/output")
```

Comment:
`${BASEOUTPUT}`?

Reply:
No, BASEOUTPUT has the reference outputs. This directory is for the outputs generated for the test runs (i.e., the outputLocation).